### PR TITLE
ruins borg blogs

### PR DESCRIPTION
**Pull request by dannno**
:cl: Dannno
tweak: Borg announcements have a longer cooldown.
/:cl:

Reasoning:
Fuck your borg blogs. Jesus christ, I don't care what every single little thought that pops into your head is. I would remove borgs being able to announce things entirely if I had the brain or the energy.

### CODE REVIEW

**Comment by kevinz000** [comment]
Yeah no fuck off silicons don't need more nerfs. If they're actually too annoying then make borgs have a longer cooldown, not the AI. If you're not going to put effort into a PR, don't fucking make it.

**Comment by dannno** [comment]
thanks kev

**Comment by sawrge** [comment]
tyhaks kev

**Comment by Shadowlight213** [comment]
Muh blorgs

**Comment by Armhulen** [comment]
tanks kev

**Comment by dannno** [comment]
thanks kev

**Comment by TechnoAlchemisto** [comment]
this is a buff so borgs can just follow their laws instead of having fun

**Comment by dannno** [comment]
fun is a buzzword

**Comment by TechnoAlchemisto** [comment]
borg players don't like fun they should all be permabanned this is a step in the right direction

**Comment by coiax** [comment]
Um, isn't the problem cyborgs being able to use communication consoles without actually being in range? If the borgs had to actually travel to a comms console to announce, then it'd be less of a problem

**Comment by dannno** [comment]
I should probably add that too but I guarantee it would not be enough on its own
These slimy fucks are desperate enough to be listened to that they would absolutely run to a console every three seconds to announce some stupid shit

I probably won't remove the range, because this isn't meant to be a nerf to borgs making announcements. It's a nerf to borgs making them every fucking three seconds about every little thing that happens. Now, borgs will have to think; "Should I use my announcement to make a funny little observation about something no one cares about, or save it in case a situation arises in which I need to inform the crew of danger?"

**Comment by dannno** [comment]
Like holy fuck, it's all bullshit they could say over the radio
just say it over the fucking radio

**Comment by kilkun** [comment]
increasing the cooldown at which borgs can scream to the whole z-level regardless of comms being up or down is hardly a nerf.

:+1: on this pr, fuck silicons.

**Comment by TehZombehz** [comment]
Just remove cyborgs tbh.

**Comment by dannno** [comment]
You play Valentine, correct?
You're one of the few borg bloggers that's going "feel the pain" from this.
I've seen a lot of people talking about how annoying it is. How annoying YOU are. Assuming I'm correct.

**Comment by kilkun** [comment]
Borgs should be seen and not heard tbqh. Like drones, but forced to serve humans.

**Comment by TehZombehz** [comment]
Glad to see that the code and the coder are basically telling me how to play this game, with the bonus of the coder telling me they think I'm annoying and don't like me.
Remind me not to roleplay, or to play at all.

**Comment by dannno** [comment]
You keep posturing like I'm the only one who hates this, but I did this because after seeing all of the complaining in deadchat people ASKED me to. You also keep acting like I'm imposing some kind of great cruelty on borg players, but you and a couple of others are the only ones who use these announcements often at all to begin with, let alone spam them as much as you do. 

Stop being so dramatic. Use your radio instead.

**Comment by TehZombehz** [comment]
6 times longer seems a little intense to me. It's still possible for AIs to spam announcements as such. Is that more okay because it's the AI and not cyborgs, or is it just not happening enough to merit a change? Will that limiter get added to AIs (or back in, according to the commits) if we start seeing a rise of announcement use by AIs (in addition to the vox)? Who decides what behavior is okay for AIs and not for cyborgs, or for cyborgs in general?
I was more okay with that 2400 you had in the first commit, at least.

**Comment by dannno** [comment]
As quite literally half of the problem that caused this PR to be a thing people felt was needed, you might be surprised and upset when you find out that I care very little about what you think about this

**Comment by kevinz000** [comment]
no one cared about you either take your salt out of here.

**Comment by dannno** [comment]
I can't actually decipher what you're trying to get at with that, sorry kev
I maintain: This is going to be an unnoticeable change for the majority of borgs, and will only piss two people that people find annoying off.

**Comment by kevinz000** [comment]
oh, and, one announcement per minute per console max with 2 starting consoles on most maps is hardly considered spam. just because you can't deal with hearing some dings doesn't mean you have to come to github to ruin things.

**Comment by PKPenguin321** [comment]
\>"silicon players"
\>metallic furries

**Comment by dannno** [comment]
I'm not the only one with a problem. I'm just the guy who made the PR at the behest of others.

**Comment by vcordie** [comment]
thanks kev

**Comment by TechnoAlchemisto** [comment]
Epic thread

**Comment by kevinz000** [comment]
thanks techno

**Comment by ExcessiveUseOfCobblestone** [comment]
>glad to see the coder and the code are telling me how to play

No, the person on an open source community is tired of seeing your blog that they literally cannot opt out of that also takes up a considerable amount of space in chat box due to how it formats.

You can opt out of Vox messages via prefs, and if AIs misused the feature they too would receive a longer cooldown.

That or we can take this to policy discussion to talk about banning silicon players for misusing their silicon ability to nearly-spam everyone's chat.

Silicon players forcing players to make silicon nerfs, really makes you think.

**Comment by Banana-Fist** [comment]
What's the point of this merge? This is just salt flying back and forth and doesn't have any objective impact on the game, just subjective whining.

**Comment by dannno** [comment]
easy way to put an end to an issue people are complaining about without affecting borgs that much
and people like TehZombies will tell you this is some kind of huge nerf to borgs but your average borg simply doesn't use announcements that often and the chances that most players will actually notice any difference is slim.

**Comment by kevinz000** [comment]
Why isn't there a policy that you have to play something to change something? I doubt you've ever played silicons long enough to need having to use announcements.

**Comment by Archie700** [comment]
Holy shit this is the most useless pr ever.

Just fucking law 2 the borg to not use the consoles jesus.

If you guys can't tolerate a bloody borg using consoles to the point that you want to pr a nerf you might as well say out you want a norp server.

This is just a pr filled with salt.

**Comment by coiax** [comment]
The branch name speaks volumes.

**Comment by dannno** [comment]
I've been playing for a long time, of course I've played silicons.

File: code/game/machinery/computer/communications.dm [hunk 1]
```diff
@@ -651,8 +653,8 @@ var/const/CALL_SHUTTLE_REASON_LENGTH = 12
 	dat += "<BR><BR>\[ [(src.aistate != STATE_DEFAULT) ? "<A HREF='?src=\ref[src];operation=ai-main'>Main Menu</A> | " : ""]<A HREF='?src=\ref[user];mach_close=communications'>Close</A> \]"
 	return dat
 
-/obj/machinery/computer/communications/proc/make_announcement(mob/living/user, is_silicon)
-	if((is_silicon && ai_message_cooldown > world.time) || (!is_silicon && message_cooldown > world.time))
+/obj/machinery/computer/communications/proc/make_announcement(mob/living/user, is_silicon, is_cyborg)
+	if((is_silicon && ai_message_cooldown > world.time) || (!is_silicon && message_cooldown > world.time) || (is_cyborg && borg_message_cooldown > world.time))
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/machinery/computer/communications.dm]: what even is is_silicon wtf

**Comment by Iamgoofball** [comment]
Can someone please explain what a Borg blog is

**Comment by coiax** [comment]
If you keep the communications console dialog open, you can make announcements even when not next to the console. Some cyborgs do this so they can make regular announcements.

You know, I have an idea.

File: code/game/machinery/computer/communications.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [inline code/game/machinery/computer/communications.dm]: also: this is the only place where is_silicon is used. WHY

Comment (dannno) [inline code/game/machinery/computer/communications.dm]: don't ask me

Comment (dannno) [inline code/game/machinery/computer/communications.dm]: if you have suggestions on what to improve that would be hella appreciated

**Comment by optimumtact** [comment]
Administrative issue more than a code one I'd say.

Come back to us when the admins start whinging about it.

**Pull request closed without merging**